org.mixedrealitytoolkit.input/Tests/Runtime/BasicInputTestsForControllerlessRig.cs

```diff
@@ -672,7 +650,7 @@ public IEnumerator GrabAnchorTest()
             Assert.IsTrue((interactable.HoveringGrabInteractors[0] as GrabInteractor).enabled, "Interactor wasn't enabled");
 
             TestUtilities.AssertAboutEqual(interactable.HoveringGrabInteractors[0].GetAttachTransform(interactable).position,
-                                           cubePos, "Grab interactor's attachTransform wasn't where we wanted it to go!", 0.001f);
+                                           cubePos, "Grab interactor's attachTransform wasn't where we wanted it to go!", 0.5f);
```

Comment:
A while half a meter is way TOOO much tolerance.   Something seems to be failing here

Reply:
I took a look at what is failing.  The problem is that the "controllerless" test has both the left and right hands hovering the cube.  There should only be one hand, the right hand.

**Worker Legacy Rig (One Hand)**

<img width="875" alt="image" src="https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/36461279/2026cf78-0086-45a2-94da-dc5759e70b18">

**Not Working New Rig (Two Hands!)**

<img width="872" alt="image" src="https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/36461279/3dcd471d-1842-4e20-814a-7828f86a7b3d">

Reply:
Got to figure out why the "left" hand is active and hovering the cube

Reply:
Found the bug!!!!!!!!!!!

It is in `HandJointInteractor.cs`...look at line 103

```cs
bool tracked = (trackedPoseDriver.GetInputTrackingState().HasPositionAndRotation() || interactionPointTracked);
```

It should be

```cs
result &= (trackedPoseDriver.GetInputTrackingState().HasPositionAndRotation() || interactionPointTracked);
```

Reply:
Unit Tests save the day!

Reply:
Restored original test value and implemented fix in e869b04ef6796f79c486e2221cce582d66adc248 , resolving.